libpod.conf

```diff
@@ -12,7 +12,8 @@ conmon_path = [
 	    "/usr/bin/conmon",
 	    "/usr/sbin/conmon",
 	    "/usr/local/bin/conmon",
-	    "/usr/local/sbin/conmon"
+	    "/usr/local/sbin/conmon",
```

Comment:
Actually, if we're in here making changes, we should move `/usr/local/*bin` paths closer to the top - it's somewhat confusing that a user-build conmon will not override a system install, and doesn't match the behavior of `runc`.

Reply:
What about discovering them via dynamic `$PATH` lookup as fallback?

Reply:
This has been discussed - I'm not really opposed, as long as config lookup comes first.